src-tauri/src/main.rs

```diff
@@ -30,30 +28,29 @@ struct DiskStats {
 #[tauri::command]
 async fn farming(path: String) -> [u8; 32] {
   let path_buf = PathBuf::from(path);
-  let path = get_path(Some(path_buf));
   let node_rpc_url: String = "ws://127.0.0.1:9944".to_string();
-  let ws_server_listen_addr: SocketAddr = "127.0.0.1:9955".parse().unwrap();
 
   // start farming, and return the public key of the farmer
-  let public_key = farm(path, &node_rpc_url, ws_server_listen_addr).await;
+  let public_key = farm(path_buf, &node_rpc_url).await;
   public_key.unwrap()
 }
 
 #[tauri::command]
 fn get_disk_stats(dir: String) -> DiskStats {
-  println!("{}", dir.to_string());
+  println!("{}", dir);
```

Comment:
Is this a debugging print?

Reply:
yes, printing to the CLI. It was there already, I did not bother changing it. I will change it to `log::info`, don't want to remove that, may become handy :)